apps/website/blog/2024-12-01-maci-getting-started.md

```diff
@@ -0,0 +1,331 @@
+:::
+
+### Download the `.zkey` files
+
+MACI has two main zk-SNARK circuits. Each circuit is parameterised. There should be one
+`.zkey` file for each circuit and set of parameters.
+
+Unless you wish to generate a fresh set of `.zkey` files, you should obtain
+them from someone who has performed a multi-party trusted setup for said
+circuits. For more details on which artifacts have undergone a trusted setup, please refer to the [Trusted Setup](/docs/security/trusted-setup) page.
+
+:::important
+Note the locations of the `.zkey` files as the CLI requires them as command-line flags.
+:::
+
+#### Download test artifacts
+
+For quickly testing MACI or trying it on a testnet we suggest using the test artifacts, using the latest dev updates you can do it by running:
+
+```bash
+pnpm download-zkeys:test
+```
+
+#### Download ceremony artifacts
+
+For production you need to use the ceremony artifacts which have undergone a trusted setup, you can download it with the command:
+
+```bash
+pnpm download-zkeys:ceremony
+```
+
+:::info
+Currently, the ceremony artifacts work with MACI version up to 2.x
+:::
+
+## Deploy Contracts
+
+Before deploying the contracts we need to do some final configurations to our repository.
+
+### Set the environment variables
+
+Head to the `packages/contracts` folder and copy the `.env.example` file.
+
+```bash
+cd packages/contracts && \
+cp .env.example .env
+```
+
+Make sure to include a mnemonic and RPC url (make sure to replace NETWORK with the network you want to use).
+
+```js
+MNEMONIC = "your ethereum secret key";
+NETWORK_RPC_URL = "the eth provider url";
+NETWORK_ETHERSCAN_API_KEY = "etherscan api key";
+```
+
+### Generate Coordinator Keys
+
+In order to encrypt and decrypt the messages, we need to generate MACI key pair, treat it just as your ethereum private keys, store it on a safe place as you won't be able to finish a round if you lose access, or a bad actor could decrypt the results and publish them online.
+
+Before deploying a Poll, make sure you have set the coordinator public MACI key to which you own the private key. You can generate a new key pair using maci-cli by running the following commands:
+
+```bash
+cd packages/cli && \
+node build/ts/index.js genMaciKeyPair
+```
+
+### Set the configuration file
+
+Head back to the contracts folder and copy the config example and update the fields as necessary:
+
+```bash
+cd ../contracts && \
+cp deploy-config-example.json deploy-config.json
+
+```
+
+#### ConstantInitialVoiceCreditProxy
+
+Defines how many credits will get each voter.
+
+| Property   | Description                                              |
+| ---------- | -------------------------------------------------------- |
+| **deploy** | Defines if the contract is needs to be deployed.         |
+| **amount** | Defines how many vote credits will get each participant. |
+
+:::important
+The current deployed contract has **99** as the voice credits amount.
```

Comment:
Also you haven't talked before about having already deployed contracts nor how to skip certain steps so this sentence might be a bit confusing

Reply:
> on which network? are they all the same?

just noticed that on testnet we have 99. and some mainnets we have 200, so I guess it's better to remove this line

Reply:
I agree with removing this sentence, since for example on the Devcon round, each person actually has 1000 voice credits. maybe you're talking about the vote.maci?

Would it be better to say "there's only constant voice credit proxy contract right now, but we plan to build more such as zupass voice credit proxy, in the future"? kind of introducing the voice credit giving out rules could be modified?